ui/src/app/services/content.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable, OnInit } from "@angular/core";
+import { LessonContent } from "../types/Content"
+import { BehaviorSubject } from "rxjs";
+import { ApiService } from "./api.service";
+
+@Injectable()
+export class ContentService {
+     getAllQuestions = true;
+     questionList: LessonContent[] = [];
+     index = 0;
+     
+     
+     constructor(private apiService: ApiService) {
+          if (this.getAllQuestions) {
```

Comment:
Confusing variable nomenclature -- seems like a function that you aren't calling.